src/modules.d.ts

```diff
@@ -112,4 +116,8 @@ declare module 'electron' {
         export function readHtml(type?: string): string;
         export function readText(type?: string): string;
     }
+
+    export namespace net {
```

Comment:
Already defined in this file https://github.com/HadoukenIO/core/blob/develop/src/modules.d.ts#L36